packages/menu/src/Menu.ts

```diff
@@ -400,6 +407,17 @@ export class Menu extends SizedMixin(SpectrumElement, { noDefaultSize: true }) {
         this.addEventListener('pointerup', this.handlePointerup);
         this.addEventListener('sp-opened', this.handleSubmenuOpened);
         this.addEventListener('sp-closed', this.handleSubmenuClosed);
+
+        // Add touch event listeners for iPad scroll detection
+        this.addEventListener('touchstart', this.handleTouchStart, {
+            passive: true,
+        });
+        this.addEventListener('touchmove', this.handleTouchMove, {
+            passive: true,
+        });
+        this.addEventListener('touchend', this.handleTouchEnd, {
```

Comment:
there is already a touchend event listener above this. should that be removed in favor of this?